integration/stability/hypervisor_stability_kill_test.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 HyperHQ Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This test will kill a running container's
+# hypervisor, and see how we react to cleanup.
+
+set -e
+
+cidir=$(dirname "$0")
+
+source "${cidir}/../../metrics/lib/common.bash"
+
+# Environment variables
+IMAGE="${IMAGE:-busybox}"
+CONTAINER_NAME="${CONTAINER_NAME:-test}"
+PAYLOAD_ARGS="${PAYLOAD_ARGS:-tail -f /dev/null}"
+
+# Set the runtime if not set already
+RUNTIME="${RUNTIME:-kata-runtime}"
+
+setup()  {
+	clean_env
+	sudo docker run --runtime=$RUNTIME -d --name $CONTAINER_NAME $IMAGE $PAYLOAD_ARGS
+	num=$(ps aux | grep ${HYPERVISOR_PATH} | grep -v grep | wc -l)
+	[[ ${num} -eq 1 ]] || die "hypervisor count:${num} expected:1"
+}
+
+killhypervisor()  {
```

Comment:
Nit: `CamelCase` or `snake_case` would make this function name easier to read.

Reply:
Updated. Thanks!